src/components/Itembox/Itembox.scss

```
@@ -0,0 +1,54 @@
+.itemList {
+  padding: 5%;
+  width: 500px;
+  height: 700px;
+  background-color: #f6f5e8;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 10px;
+  &:hover {
+    background-color: #f0efe1;
+  }
+  .itemImg {
+    height: 60%;
+  }
+  .itemName {
+    display: flex;
+    height: 10%;
+    justify-content: center;
+    align-items: center;
+  }
```

```suggestion
  .itemImg {
    height: 60%;
  }
  
  .itemName {
    display: flex;
    height: 10%;
    justify-content: center;
    align-items: center;
  }
```

위코드에서 제공하는 코드 컨벤션입니다. css코드를 작성할 때 선택자와 속성을 작성하는 곳 사이에는 빈 공백을 한 줄 추가해 가독성을 높여주세요